scripts/run-ui-tests

```diff
@@ -14,7 +14,9 @@ source "$self_dir"/utils.sh
 intellij_version=${1-}
 test_method_prefix=${2-}
 
-export IDEPROBE_DISPLAY=xvfb
+if [ -n "${CI:-}" ]; then
+    $CI && export IDEPROBE_DISPLAY=xvfb
```

Comment:
Maybe just for safety I wouldn't execute whatever the value of `CI` variable is... instead just

```shell script
if [[ ${CI-} == true ]]; then
  export IDEPROBE_DISPLAY=xvfb
fi
```